src/Documentation/Getting-Started-With-Orleans/Debugging.md

```diff
@@ -18,6 +21,8 @@ Add it to the list of symbol servers in Visual Studio in Tools/Options/Debugging
 Make sure there is a trailing slash in the URL.
 Visual Studio 2015 has a bug with parsing it.
 
-## Sources
+### Sources
 
 You can download zipped sources for specific releases of Orleans from the [Releases page](https://github.com/dotnet/orleans/releases).
+However, due to the LF/CR differences between Windows and Unix (default for GitHub), debugger may complain about a mismatch of the sources and symbols.
+The workaround is to check out the corresponding version tag on a Windows machine to get sources with the matching LF/CR ending.
```

Comment:
This seems unfortunate - maybe we need to change the git config on the build agents used to create the packages?